docs/api/README.md

```diff
@@ -113,17 +115,36 @@ If you add a `target="_blank"` to your `a` element, you must omit the `@click="n
   <router-link :to="{ path: 'relative/path'}" append></router-link>
   ```
 
-### tag
+### tag (3.1.0 < 4.0.0)
```

Comment:
can you remove all changes after this one (including it). The docs of v4 are at https://next.router.vuejs.org/

Reply:
Well, I feel silly now. I didn't find that link anywhere yesterday :-P